agent-install/agent-install.sh

```diff
@@ -94,7 +94,11 @@ Additional Edge Device Variables (in environment or config file):
     AGENT_WAIT_MAX_SECONDS: Maximum seconds to wait for the Horizon agent to start or stop. Default: 30
 
 Additional Edge Cluster Variables (in environment or config file):
+<<<<<<< HEAD
```

Comment:
This is a merge conflict. Please resolve it by keeping what is in the HEAD and update the PR

Reply:
Done